docs/static/ingest-convert.asciidoc

```diff
@@ -0,0 +1,87 @@
+[[ingest-converter]]
+=== Converting Ingest Node Pipelines
+
+After implementing {ref}/ingest.html[ingest] pipelines to parse your data, you
+might decide that you want to take advantage of the richer transformation
+capabilities in Logstash. For example, you may need to use Logstash instead of
+ingest pipelines if you want to:
+
+* Ingest from more inputs. Logstash can natively ingest data from many other
+sources like TCP, UDP, syslog, and relational databases.
+
+* Use multiple outputs. Ingest node was designed to only support Elasticsearch
+as an output, but you may want to use more than one output. For example, you may
+want to archive your incoming data to S3 as well as indexing it in
+Elasticsearch.
+
+* Take advantage of the richer transformation capabilities in Logstash, such as
+external lookups.
+
+* Use the persistent queue feature to handle spikes when ingesting data (from
+Beats and other sources).
+
+To make it easier for you to migrate your configurations, Logstash provides an
+ingest pipeline conversion tool. The conversion tool takes the ingest pipeline
+definition as input and, when possible, creates the equivalent Logstash
+configuration as output. 
+
+See <<ingest-converter-limitations>> for a full list of tool limitations.
+
+[[ingest-converter-run]]
+==== Running the tool
+
+You'll find the conversion tool in the `bin` directory of your Logstash
+installation. See <<dir-layout>> to find the location of `bin` on your system.
+
+To run the conversion tool, use the following command:
+
+[source,shell]
+-----
+bin/ingest-convert.sh --input INPUT_FILE_URI --output OUTPUT_FILE_URI [--append-stdio]
```

Comment:
@dedemorton maybe it's worth mentioning that at present only `file://` type URIs are supported here?